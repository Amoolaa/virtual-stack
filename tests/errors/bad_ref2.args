"tests/errors/bad_ref2.x2017"
