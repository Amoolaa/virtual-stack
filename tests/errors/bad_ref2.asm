FUNC LABEL 0
    MOV STK A VAL 69
    REF VAL 3 STK A
    PRINT STK A
    RET
