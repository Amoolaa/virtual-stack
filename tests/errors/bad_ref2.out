REF: arguments cannot be VAL typed.
